.github/CONTRIBUTING.md

```diff
@@ -1,59 +1,79 @@
 # TERArium Contributing Guide
 
-Hi! We're really excited that you're interested in contributing to TERArium! Before submitting your contribution, please read through the following guide.
+Hi! We're really excited that you're interested in contributing to TERArium! Before submitting your contribution, please
+read through the following guide.
 
 ## Repository Setup
 
-The TERArium repo is a mixed repo using yarn workspaces and Java backend components. The package managers used to install and link dependencies are [yarn](https://yarnpkg.com/getting-started) and [gradle](https://gradle.org/).
+The TERArium repo is a mixed repo using yarn workspaces and Java backend components. The package managers used to
+install and link dependencies are [yarn](https://yarnpkg.com/getting-started) and [gradle](https://gradle.org/).
 
 To develop and test the core application:
 
-1. Stand up the Gatweay servers by running the `gateway.sh up` script from within the `kubernetes/local` directory.
-2. Start up the Backend Quarkus server using `./gradlew quarkusDev` or `quarkus dev` command
-3. Start the front end by doing `yarn install` followed by `yarn workspaces webapp dev`
-4. Navigate your browser to `localhost:8078` 
+1. Stand up the TERArium services by running the `deploy-terarium.sh up` script from within the
+	 [Orchestration](https://github.com/DARPA-ASKEM/orchestration) repository
+2. Stop the `hmi-server` and `webapp` pods
```

Comment:
You should add the command to do so.

Reply:
And edit the local deployment to have a replica of `0` to easily kill them

Reply:
fixed